docs/reference/api/drivers/SAI.md

```diff
@@ -0,0 +1,52 @@
+## Serial Audio Interface (SAI)
+
+The Serial Audio Interface (SAI) provides a class to communicate with audio devices. SAI is a broader feature that encompasses audio interfaces such as I2S.
+
+SAI uses a four wire serial protocol consisting of:
+  * Master Clock Pin  (mclk)
+  * Data Pin          (sd)
+  * Bit Clock pin     (bclk)
+  * Word Clock Pin    (wclk)
+
+The master clock is used to optionally synchronize the device's IO lines. When using SAI to communicate in loopback or with an on-target audio codec, there may be an internally shared master clock and this pin is not required. The `mclk_source` field in the format object that may be optionally passed into the SAI constructor can be set to internal, external or sibling.
```

Comment:
This section starts off well explaining the Master Clock, but the next sentence about the pin, is confusing - which pin? The Master Clock pin or the Data pin?  If the former, then it doesn't explain what the Data pin is used for at all.  This section explains the Master Clock, Bit Clock and Word Clock but should also explain what the pins themselves are used for or how the SAI uses them, and it needs to say something about the Data Pin.

Reply:
The Master Clock pin is not required to transfer data.

The data are sent on the _data pin_ using the _bit clock_ to identify when the data should be sampled.
the _word clock_ usually serves as word delimiter.

Reply:
This needs to be clarified better.